interface/main/calendar/add_edit_event.php

```diff
@@ -116,9 +126,16 @@
     $collectthis = $collectthis[array_keys($collectthis)[0]]["rules"];
 }
 ?>
+<?php $disabled = (!$edit && $GLOBALS['enable_group_therapy'] )?' disabled=true; ':'';?>
+<?php if($disabled) echo '<script>$( document ).ready(function(){
```

Comment:
@oshriromatrix 
Hi Rod,
in the whole of the code we use globals, the code works fine after hard testing group.
is this a problem?